test/bigtest/network/configs/budgets.js

```diff
@@ -10,6 +10,8 @@ const configBudgets = server => {
       ? schema.budgets.find(request.params.id).attrs
       : null;
   });
+
+  server.put(`${BUDGETS_API}/:id`, () => null);
```

Comment:
could you please implement actual `put`? for reference https://github.com/folio-org/ui-invoice/blob/master/test/bigtest/network/configs/lines.js#L14

Reply:
Updated